core/lib/multivm/src/versions/vm_fast/version.rs

```diff
@@ -0,0 +1,28 @@
+use crate::{vm_latest::MultiVMSubversion, VmVersion};
+
+#[derive(Debug, Copy, Clone)]
+pub(crate) enum FastVMVersion {
```

Comment:
Bikeshedding: `FastVmVersion` seems more in line with Rust naming conventions.

Reply:
I've also renamed other types